drivers/rtc/rtc_rv8263.c

```diff
@@ -529,27 +512,28 @@ static int rv8263c8_alarm_get_time(const struct device *dev, uint16_t id, uint16
 		return err;
 	}
 
-	if (value[0] <= MAX_SEC) {
```

Comment:
Please also remove the `MAX_x` and `MIN_x` defines. They are no longer used and could misleading.